stock_picking_batch_split/wizards/stock_picking_batch_split_wizard.py

```diff
@@ -0,0 +1,15 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 (http://www.gnu.org/licenses/agpl-3.0.html)
+
+from odoo import models, fields
+
+class StockPickingBatchSplit(models.TransientModel):
+    _name="stock.picking.batch.split.wizard"
+    _description="Split picking"
+
+    product_id = fields.Many2one("product.product", required=True)
+    batch_id = fields.Many2one("stock.picking.batch")
+
+    def split(self):
```

Comment:
Something would have to be done so that when it finds a delivery note that only has one movement and that is what we want to divide, it jumps to the next one.